app/serializers/v2/rule_serializer.rb

```diff
@@ -4,5 +4,7 @@ module V2
   # JSON serialization for an OpenSCAP Rule
   class RuleSerializer < V2::ApplicationSerializer
     attributes :ref_id, :title, :rationale, :description, :severity, :precedence
+
+    complex_attribute :remediation_issue_id, profiles: [:ref_id], security_guide: [:ref_id]
```

Comment:
What about naming it `dependent_attribute`?

Reply:
Longer 😁

Reply:
I'd also lean towards `dependent_attribute`. It better describes the use.

Reply:
What about `derived_`?

Reply:
`derived_` works for me :+1:

Reply:
:+1: